addons/point_of_sale/static/src/js/models.js

```diff
@@ -1796,7 +1796,7 @@ exports.Orderline = Backbone.Model.extend({
     can_be_merged_with: function(orderline){
         var price = parseFloat(round_di(this.price || 0, this.pos.dp['Product Price']).toFixed(this.pos.dp['Product Price']));
         var order_line_price = orderline.get_product().get_price(orderline.order.pricelist, this.get_quantity());
-        order_line_price = orderline.compute_fixed_price(order_line_price);
+        order_line_price = round_di(orderline.compute_fixed_price(order_line_price), this.pos.currency.decimals);
```

Comment:
I don't understand why do you need to round what is returned by the function 'compute_fixed_price', when I read it, I see that if there is a fiscal position it calls compute_all that already round it. And if you don't have fiscal position (probably your case), it just returns you what you gave.

So your issue is probably come from the line above that call get_price, and you should probably call round_pr in the fuunction 'get_price'. And look at the behavior of the function "get_product_price" in python which is supposed to do the same thing

Reply:
@pimodoo because of this comment, do you think I should call the rounding function somewhere else or is it good as it stands? Any help is appreciated.